set.mm

```diff
@@ -61362,6 +61424,22 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
   ffrn $p |- ( F : A --> B -> F : A --> ran F ) $=
     ( wf wfn crn ffn dffn3 sylib ) ABCDCAEACFCDABCGACHI $.
 
+  $( Characterization of a function with domain and codomain (essentially using
+     that the range is always included in the codomain).  Generalization of
+     ~ ffrn .  (Contributed by BJ, 21-Sep-2024.) $)
```

Comment:
```suggestion
     ~ ffrn .  (Contributed by BJ and AV, 21-Sep-2024.) $)
```
Or only BJ, or only AV, as you prefer: you did the work and I only reformulated.

Reply:
The proof wasn't much work - therefore no change of contributor() necessary.